src/scripts/main.js

```diff
@@ -1,3 +1,94 @@
 'use strict';
 
-// write code here
+const url
+  = `https://mate-academy.github.io/phone-catalogue-static/api/phones.json`;
+const details
+  = `https://mate-academy.github.io/phone-catalogue-static/api/phones/`;
+
+function getPhones(link) {
+  return new Promise((resolve, reject) => {
+    fetch(link)
+      .then((obj) => obj.json())
+      .then((res) => {
+        resolve(res);
+      });
```

Comment:
```suggestion
      .then((obj) => resolve(obj.json()))
```
do you need additional then ?